### ISSUE TITLE
Accessibility: Missing alt text for image

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/Ottawa-ckan#26]
CAG 2.0 Success Criteria 1.1.1 - Non text Content Level (A) Compliance violation - All images must have equivalent alternative text.
1. Go to: data.ottawa.ca
2. Observe for alt text on the image associated with the 'Map Quest' icon at the bottom of the map images.
![mapquest icon_alt text missing](https://cloud.githubusercontent.com/assets/5271253/8011350/f7808062-0b85-11e5-908d-8c7bf1e55f30.JPG)

Resolution: Add alternative text to images.
